subjects/printifnot/README.MD

```diff
@@ -0,0 +1,45 @@
+## printifnot
+
+### Instructions
+
+Write a function that takes a `string` as an argument and returns the letter `G` if the argument length is less than 3, otherwise returns **Invalid Input\n**.
+- If it's an empty string return `G` followed by a newline.
+
+### Expected function
+
+```go
+func PrintIfNot(str string) string {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"piscine"
+)
+
+func main() {
+	fmt.Print(piscine.PrintIfNot("abcdefz"))
+	fmt.Print(piscine.PrintIfNot("T2d97d1"))
+	fmt.Print(piscine.PrintIfNot(""))
+	fmt.Print(piscine.PrintIfNot("14"))
```

Comment:
I would suggest to add one case where we have a string with 3 characters.

Reply:
fixed